bin/pwa.dart

```diff
@@ -103,24 +103,24 @@ Future _generateManifestJson(String webDir, Map pubspec) async {
   await _updateIfNeeded(indexHtmlFile.path, indexHtmlContent);
 }
 
-/// If build/web is empty, run `pub build`.
+/// If build/ is empty, run `webdev build`.
 Future _buildProjectIfEmptyOrOld(List<String> sources, List<String> excludes,
     List<String> offlineDirs) async {
   // This works only with the default value.
-  if (offlineDirs.length == 1 && offlineDirs.first == 'build/web') {
-    Directory dir = new Directory('build/web');
+  if (offlineDirs.length == 1 && offlineDirs.first == 'build') {
+    Directory dir = new Directory('build');
     if (dir.existsSync() && dir.listSync().isNotEmpty) {
       int lastSourceTimestamp = _getLastTimestamp(sources, excludes);
       int lastOfflineTimestamp = _getLastTimestamp(offlineDirs, null);
       if (lastSourceTimestamp < lastOfflineTimestamp) return;
     }
-    print('Running pub build:');
-    String executable = 'pub';
+    print('Running webdev build:');
+    String executable = 'webdev';
     if (Platform.isWindows) {
       try {
-        final pr = await Process.run('pub.exe', ['--version']);
+        final pr = await Process.run('webdev.exe', ['--version']);
```

Comment:
`.exe` doesn't make sense here, If targeting windows it should be `webdev.bat` or simply `webdev` to let the platform figure out.